### PR TITLE
Jv rox 12259 push collector builder images in osci

### PR DESCRIPTION
**Pull request by JoukoVirtanen**
Add a OSCI job to push collector builder images. See https://github.com/stackrox/collector/pull/795

### CODE REVIEW

**Comment by robbycochran** [comment]
This change will add another test step that has to be created and that might slow down rehearsals and the overall build. -- why not include the builder push in the already existing push image step?

**Comment by JoukoVirtanen** [comment]
> 

Done

File: ci-operator/step-registry/stackrox/collector/integration-tests/stackrox-collector-integration-tests-workflow.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ workflow:
       - ref: stackrox-collector-integration-tests-setup
     test:
       - as: push-images
-        commands: .openshift-ci/jobs/push-images/push-images.sh
+        commands: .openshift-ci/jobs/push-images/push-images.sh collector
```
Comment (Molter73) [inline ci-operator/step-registry/stackrox/collector/integration-tests/stackrox-collector-integration-tests-workflow.yaml:8]: I think this change might be outdated, I can't see where the first parameter is used in the script.

**Pull request merged**